tests/test_typescript_features.py

```
@@ -0,0 +1,524 @@
+        throw new Error("Failed");
+    }
+    const data = await response.text();
+    return data;
+}
+"""
+
+
+def run_tldr(args: list[str], cwd: str = None) -> dict:
+    """Run tldr command and return parsed JSON output."""
+    cmd = ["tldr"] + args + ["--lang", "typescript"]
+    result = subprocess.run(cmd, capture_output=True, text=True, cwd=cwd)
```

Ruff (0.14.11)

`subprocess` call: check for execution of untrusted input

(S603)